// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`extensions.tx (provider=cockroachdb) extended client in tx can rollback via custom call 1`] = `

Invalid \`prisma.user.create()\` invocation:


Unique constraint failed on the fields: (\`email\`)
`;

exports[`extensions.tx (provider=cockroachdb) extended client in tx can rollback via normal call 1`] = `

Invalid \`xprisma.user.create()\` invocation in
/client/tests/functional/extensions/tx.ts:0:0

  XX     lastName: 'Smith',
  XX   },
  XX }),
→ XX xprisma.user.create(
Unique constraint failed on the fields: (\`email\`)
`;

exports[`extensions.tx (provider=mongodb) extended client in tx can rollback via custom call 1`] = `

Invalid \`prisma.user.create()\` invocation:


Unique constraint failed on the constraint: \`User_email_key\`
`;

exports[`extensions.tx (provider=mongodb) extended client in tx can rollback via normal call 1`] = `

Invalid \`xprisma.user.create()\` invocation in
/client/tests/functional/extensions/tx.ts:0:0

  XX     lastName: 'Smith',
  XX   },
  XX }),
→ XX xprisma.user.create(
Unique constraint failed on the constraint: \`User_email_key\`
`;

exports[`extensions.tx (provider=mysql) extended client in tx can rollback via custom call 1`] = `

Invalid \`prisma.user.create()\` invocation:


Unique constraint failed on the constraint: \`User_email_key\`
`;

exports[`extensions.tx (provider=mysql) extended client in tx can rollback via normal call 1`] = `

Invalid \`xprisma.user.create()\` invocation in
/client/tests/functional/extensions/tx.ts:0:0

  XX     lastName: 'Smith',
  XX   },
  XX }),
→ XX xprisma.user.create(
Unique constraint failed on the constraint: \`User_email_key\`
`;

exports[`extensions.tx (provider=mysql,providerFlavor=js_planetscale) extended client in tx can rollback via custom call 1`] = `target: tests.0.primary: vttablet: rpc error: code = AlreadyExists desc = Duplicate entry 'jane@smith.com' for key 'User.User_email_key' (errno 1062) (sqlstate 23000) (CallerID: userData1): Sql: "insert into \`User\`(id, email, firstName, lastName) values (:vtg1 /* VARCHAR */, :vtg2 /* VARCHAR */, :vtg3 /* VARCHAR */, :vtg4 /* VARCHAR */)", BindVars: {vtg1: "type:VARCHAR value:\\"clm0k9hfk000a4pgdsl5kzad4\\""vtg2: "type:VARCHAR value:\\"jane@smith.com\\""vtg3: "type:VARCHAR value:\\"Jane\\""vtg4: "type:VARCHAR value:\\"Smith\\""} (errno 1062) (sqlstate 23000) during query: INSERT INTO \`tests\`.\`User\` (\`id\`,\`email\`,\`firstName\`,\`lastName\`) VALUES ('clm0k9hfk000a4pgdsl5kzad4','jane@smith.com','Jane','Smith')`;

exports[`extensions.tx (provider=mysql,providerFlavor=js_planetscale) extended client in tx can rollback via normal call 1`] = `target: tests.0.primary: vttablet: rpc error: code = AlreadyExists desc = Duplicate entry 'jane@smith.com' for key 'User.User_email_key' (errno 1062) (sqlstate 23000) (CallerID: userData1): Sql: "insert into \`User\`(id, email, firstName, lastName) values (:vtg1 /* VARCHAR */, :vtg2 /* VARCHAR */, :vtg3 /* VARCHAR */, :vtg4 /* VARCHAR */)", BindVars: {vtg1: "type:VARCHAR value:\\"clm0k9h8000034pgd1r7fhxba\\""vtg2: "type:VARCHAR value:\\"jane@smith.com\\""vtg3: "type:VARCHAR value:\\"Jane\\""vtg4: "type:VARCHAR value:\\"Smith\\""} (errno 1062) (sqlstate 23000) during query: INSERT INTO \`tests\`.\`User\` (\`id\`,\`email\`,\`firstName\`,\`lastName\`) VALUES ('clm0k9h8000034pgd1r7fhxba','jane@smith.com','Jane','Smith')`;

exports[`extensions.tx (provider=mysql,providerFlavor=vitess_8) extended client in tx can rollback via custom call 1`] = `

Invalid \`prisma.user.create()\` invocation:


Unique constraint failed on the (not available)
`;

exports[`extensions.tx (provider=mysql,providerFlavor=vitess_8) extended client in tx can rollback via normal call 1`] = `

Invalid \`xprisma.user.create()\` invocation in
/client/tests/functional/extensions/tx.ts:0:0

  XX     lastName: 'Smith',
  XX   },
  XX }),
→ XX xprisma.user.create(
Unique constraint failed on the (not available)
`;

exports[`extensions.tx (provider=postgresql) extended client in tx can rollback via custom call 1`] = `

Invalid \`prisma.user.create()\` invocation:


Unique constraint failed on the fields: (\`email\`)
`;

exports[`extensions.tx (provider=postgresql) extended client in tx can rollback via normal call 1`] = `

Invalid \`xprisma.user.create()\` invocation in
/client/tests/functional/extensions/tx.ts:0:0

  XX     lastName: 'Smith',
  XX   },
  XX }),
→ XX xprisma.user.create(
Unique constraint failed on the fields: (\`email\`)
`;

exports[`extensions.tx (provider=sqlite) extended client in tx can rollback via custom call 1`] = `

Invalid \`prisma.user.create()\` invocation:


Unique constraint failed on the fields: (\`email\`)
`;

exports[`extensions.tx (provider=sqlite) extended client in tx can rollback via normal call 1`] = `

Invalid \`xprisma.user.create()\` invocation in
/client/tests/functional/extensions/tx.ts:0:0

  XX     lastName: 'Smith',
  XX   },
  XX }),
→ XX xprisma.user.create(
Unique constraint failed on the fields: (\`email\`)
`;

exports[`extensions.tx (provider=sqlserver) extended client in tx can rollback via custom call 1`] = `

Invalid \`prisma.user.create()\` invocation:


Unique constraint failed on the constraint: \`dbo.User\`
`;

exports[`extensions.tx (provider=sqlserver) extended client in tx can rollback via normal call 1`] = `

Invalid \`xprisma.user.create()\` invocation in
/client/tests/functional/extensions/tx.ts:0:0

  XX     lastName: 'Smith',
  XX   },
  XX }),
→ XX xprisma.user.create(
Unique constraint failed on the constraint: \`dbo.User\`
`;
